src/main/java/com/petproj/moneychecker/mappers/UserMapper.java

```diff
@@ -0,0 +1,16 @@
+package com.petproj.moneychecker.mappers;
+
+import com.petproj.moneychecker.dto.UserDTO;
+import com.petproj.moneychecker.entities.User;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.mapstruct.factory.Mappers;
+
+@Mapper
+public interface UserMapper {
+    UserMapper INSTANCE = Mappers.getMapper(UserMapper.class);

```

Comment:
Are you sure you need this instance here? Will it work if we remove this INTANCE from here?